action.yml

```
@@ -62,6 +62,11 @@ runs:
         username: ${{ inputs.registry_username }}
         password: ${{ inputs.registry_token }}
 
+    - name: Set current date as env variable
```

```suggestion
    - name: Set the current date as a step variable
```
Just a nitpick, but I would say, that an env variable is something else.